src/components/chat/ChatModule.css

```
@@ -0,0 +1,250 @@
+.bubble {
+    max-width: 60%;
+    width: fit-content;
+    padding: 0.5rem 1rem;
+    border-radius: 15px;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    font-size: 0.9rem;
+    overflow: hidden;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)